src/main/java/helpers/RequestBuilder.java

```diff
@@ -0,0 +1,18 @@
+package helpers;
+
+public class RequestBuilder {
+    private final static String PROJECT_NAME = "superadmin_personal";
+    private final static String API_VERSION = "api/v1/";
+
+    public static String urlBuilder(String endpoint) {
+        return getApiVersion() + getProjectName() + endpoint;
+    }
+
+    public static String getApiVersion() {
```

Comment:
Used in this class only. No need to create getter.

Reply:
removed